pyjobs/api/tests/test_job_resources.py

```diff
@@ -0,0 +1,62 @@
+from json import loads
+from unittest.mock import patch
+
+from django.shortcuts import resolve_url
+from django.test import TestCase
+from model_mommy import mommy
+
+from pyjobs.core.models import Job
+
+
+class TestJobResourceList(TestCase):
+
+    @patch('pyjobs.core.models.post_telegram_channel')
+    def setUp(self, _mocked_post_telegram_channel):
+        self.jobs = mommy.make(Job, _quantity=21)
+        url = resolve_url('api_dispatch_list', resource_name='jobs')
+        self.response = self.client.get(url)
+        self.response.text = self.response.content.decode('utf-8')
+        self.response.json = loads(self.response.text)
+
+    def test_status(self):
+        self.assertEqual(200, self.response.status_code)
+
+    def test_pagination(self):
+        self.assertEqual(self.response.json['meta']['limit'], 20)
+        self.assertEqual(self.response.json['meta']['total_count'], 21)
+        self.assertTrue(self.response.json['meta']['next'])
+
+    def test_contents(self):
+        for job in self.jobs[:20]:  # skip jobs beyond the first page
+            with self.subTest():
+                self.assertIn(job.company_email, self.response.text)
+
+
+class TestJorResourceDetail(TestCase):
```

Comment:
Trocar nome do testcase para `TestJobResourceDetail`

Reply:
Done in 07fe1de3f07f1b184404695ada422e72d2181282